eve/workers/master/Dockerfile

```diff
@@ -46,9 +45,7 @@ ENV HOME /home/eve
 # Setup nodejs environmnent
 #
 
-ENV CXX=g++-4.9
-ENV LANG C.UTF-8
```

Comment:
I would keep the `ENV LANG C.UTF-8` by experience we had issue in the CI where the mocha command would fail if this var is not set

Reply:
@tcarmet It's there but it's duplicated - it was two times on the code

Reply:
oh my bad